packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -84,7 +85,20 @@ const TradersHub = () => {
             >
                 {can_show_notify && <Notifications />}
                 <div id='traders-hub' className='traders-hub' ref={traders_hub_ref}>
-                    <WalletsBanner />
+                    <WalletHeader account_type='demo' />
+                    <WalletHeader currency='AUD' shortcode='svg' />
+                    <WalletHeader currency='EUR' shortcode='svg' is_open_wallet={true} />
+                    <WalletHeader currency='USD' shortcode='malta' />
+                    <WalletHeader currency='BTC' shortcode='svg' />
+                    <WalletHeader currency='ETH' shortcode='svg' />
+                    <WalletHeader currency='USDT' shortcode='svg' />
+                    <WalletHeader currency='eUSDT' shortcode='svg' />
+                    <WalletHeader currency='tUSDT' shortcode='svg' />
+                    <WalletHeader currency='LTC' shortcode='svg' />
+                    <WalletHeader currency='USDC' shortcode='svg' />
+                    <WalletHeader account_status='pending' currency='ETH' shortcode='svg' />
+                    <WalletHeader account_status='need_verification' currency='EUR' shortcode='malta' />
+                    <WalletHeader account_status='failed' currency='USD' shortcode='svg' />
```

Comment:
@sergei-deriv is this something to be refactored later or are you waiting for something else to be merged ?.

Maybe we can do it something like this instead of repeating it so many times

`const wallets = [
    { account_type: 'demo' },
    { currency: 'AUD', shortcode: 'svg' },
    { currency: 'EUR', shortcode: 'svg', is_open_wallet: true },
    { currency: 'USD', shortcode: 'malta' },
    { currency: 'BTC', shortcode: 'svg' },
    { currency: 'ETH', shortcode: 'svg' },
    { currency: 'USDT', shortcode: 'svg' },
    { currency: 'eUSDT', shortcode: 'svg' },
    { currency: 'tUSDT', shortcode: 'svg' },
    { currency: 'LTC', shortcode: 'svg' },
    { currency: 'USDC', shortcode: 'svg' },
    { account_status: 'pending', currency: 'ETH', shortcode: 'svg' },
    { account_status: 'need_verification', currency: 'EUR', shortcode: 'malta' },
    { account_status: 'failed', currency: 'USD', shortcode: 'svg' },
];`

`const walletHeaders = wallets.map(wallet => <WalletHeader {...wallet} />);`

Something like this maybe ? 👀

Reply:
@thisyahlen-deriv this is just for testing purpose. Of course nobody will use WalletHeader like this. WalletHeader is just part of Wallet and this is just UI part of desktop wallet

Reply:
Alrightyy then 🙏